recipes/elfutils/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "elfutils" %}
+{% set version = "0.167" %}
+{% set sha256 = "3f300087c42b6f35591163b48246b4098ce39c4c6f5d55a83023c903c5776553" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://fedorahosted.org/releases/e/l/elfutils/{{ version }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - toolchain
+    - zlib 1.2.*
+    - m4
+
+  run:
+    - zlib 1.2.*
+
+about:
+  home: https://fedorahosted.org/elfutils/
+  license: LGPL-3.0
```

Comment:
Is there a license file we can include from the source using `license_file`?

Also can add `license_family: LGPL`.